net/strongswan/files/swanctl.init

```diff
@@ -433,13 +432,6 @@ config_remote() {
 
 	[ "$gateway" = "any" ] && remote_gateway="%any" || remote_gateway="$gateway"
 
-	[ -z "$local_gateway" ] && {
```

Comment:
Actually, if `local_gateway` doesn't get set anywhere, then this is *always* true so this is far from "dead".